src/status_im/contexts/communities/events.cljs

```diff
@@ -163,12 +163,15 @@
 (defn initialize-permission-addresses
   [{:keys [db]} [community-id]]
   (when community-id
-    (let [accounts        (get-in db [:wallet :accounts])
-          sorted-accounts (sort-by :position (vals accounts))
-          addresses       (set (map :address sorted-accounts))]
+    (let [accounts         (get-in db [:wallet :accounts])
+          sorted-accounts  (sort-by :position (vals accounts))
+          watched-accounts (remove #(:watch-only? %) sorted-accounts)
```

Comment:
generally is best to `filter` first and after sort, since `filter` is `O(n)` while sorting is at least `O(n * logn)`